feature/repair/src/main/java/com/iteneum/repair/data/RepairViewModel.kt

```diff
@@ -0,0 +1,80 @@
+package com.iteneum.repair.data
+
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.setValue
+import androidx.core.net.toUri
+import androidx.core.text.isDigitsOnly
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import com.iteneum.RepairRequest
+import com.iteneum.UIEventRepair
+import com.iteneum.network.DataState
+import dagger.hilt.android.lifecycle.HiltViewModel
+import kotlinx.coroutines.launch
+import javax.inject.Inject
+
+/**
+ * File that contains [RepairViewModel] that works as ViewModel for the Repair View.
+ * It will handle the data work & process when loading & be done with the view.
+ *
+ * @author Jose Miguel Garcia Reyes
```

Comment:
Should be only 1 author, change it for modifiedBy

Reply:
Ok, updated :)